src/main/java/org/freeswitch/esl/client/outbound/OutboundChannelInitializer.java

```diff
@@ -12,7 +14,8 @@
 public class OutboundChannelInitializer extends ChannelInitializer<SocketChannel> {
 
     private final IClientHandlerFactory clientHandlerFactory;
-    private ExecutorService callbackExecutor = Executors.newSingleThreadExecutor();
+    private ExecutorService callbackExecutor = Executors.newFixedThreadPool(2);
```

Comment:
You can change the `callbackExecutor` in `OutboundChannelInitializer` with the method `setCallbackExecutor` outside whatever you want, without modify the `OutboundChannelInitializer`.

Reply:
I wanted to modify the default `callbackExecutor` from `SingleThreadExecutor` to `FixedThreadPool(2)` for default execution by `OutboundClientHandler`, one being for handling `ESL` events and another for `command-response`. This is default behavior change hence changed here. Someone can still override and set it to much larger pool based on use-case by calling the `setCallbackExecutor`. 

Incase you think this should be still `newSingleThreadExecutor` and `OutboundClientHandler` should all times override this, then I can make that change.

Reply:
@kingster Whenever you created a different EventExecutorGroup for the handler you don't need to make it two for the command-response messages. It will be handled by EventExecutorGroup thread